File: projects/03. Software_Technologies - july 2018/30. MY_EXAM/JS ExpressJS Overview - IssueTracker/models/issue.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    const Issue = sequelize.define("Issue", {
        title: {
            type: Sequelize.TEXT
        },
        content: {
            type: Sequelize.TEXT
        },
        priority: {
            type: Sequelize.INTEGER
        }
    }, {timestamps: false})

    return Issue;

};

File: projects/Software_Technologies/CSharp/MY_EXAM/IssueTracker/IssueTracker/Models/Issue.cs
namespace IssueTracker.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Issue
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public string Title { get; set; }

        [Required]
        public string Content { get; set; }

        [Required]
        public double Priority { get; set; }
    }
}
